### DISCUSSION TITLE
database of user settings question

### DISCUSSION THREAD

**Discussion by jermacalocas** [supermerill/SuperSlicer#2677] [Q&A] [answered]
i want to use superslicer on multiple computers but have the settings come with me. is there a way to do this without having to manually  copy and update the /user/roaming/superslicer folder each time i move from desktop to laptop. 

the only thing ive found is to setup a windows directory server on my network and move my user data to that so i can log into each computer in the house and have my settings come with me. 

i am hoping to find a way to ask the program, via settings, where the user info gets stored. this when i move the flash drive to each computer the settings are located within the flash drive while also not in the superslicer program folder so it can still be easily updated. you would need to update the settings location with each update, but that would just take a close and reopen to pull in all the settings.

or am i overlooking an obvious setting.

**Comment by neophyl** [comment, marked as answer]
There already a question about this here as well as several issues. https://github.com/supermerill/SuperSlicer/discussions/1772

**Reply by jermacalocas** [reply]
yup, I dropped the ball and asked before looking. Found that thread and got is fixed.

thank you for your reply
